docs/core/tools/project-json.md

```diff
@@ -312,6 +341,106 @@ For example:
     }
 ```
 
+### version
+Type: String
+
+Specifies the version or version range of the dependency. Use the \* wildcard to specify a 
+[floating dependency version](https://docs.nuget.org/ndocs/consume-packages/dependency-resolution#floating-versions).
+
+For example:
+
+```json
+"dependencies": { 
+    "Newtonsoft.Json": { 
+        "version": "9.0.1" 
+    }
+}
+```
+
+### type
+Type: String
+
+Specifies the type of the dependency. It can be one of the following values: `default`, `build` or `platform`. The default value is `default`.
```

Comment:
@emgarten can you please help answering this? what does "type": "default" means in the dependencies node of a project.json file?